src/us/kbase/workspace/test/controllers/handle/HandleServiceController.java

```diff
@@ -140,6 +140,11 @@ private File createHandleServiceDeployCfg(
 		hs.add("mongo-host", "127.0.0.1");
 		hs.add("mongo-port", "" + mongo.getServerPort());
 		hs.add("mongo-database", mongoDB);
+		hs.add("mongo-user", "");
+		hs.add("mongo-password", "");
+
+		hs.add("start-local-mongo", 1);
```

Comment:
What does this mean?

Reply:
old HS has SQL started locally. I had an impression that mongo server will start locally as well. But Keith said mongo will be remotely hosted. So I added this config if set, mongo will start locally. Otherwise, it won't start local mongo.

Reply:
But the tests already have a mongod running at this point - the passed in MongoController manages that, and that's the server that mongo.getServerPort() returns. Another mongo server shouldn't be necessary.

Reply:
👍